sdk/core/azure-core/test/ut/operation_test.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/context.hpp>
+#include <azure/core/nullable.hpp>
+#include <azure/core/operation.hpp>
+#include <azure/core/operation_status.hpp>
+#include <azure/core/response.hpp>
+
+using namespace Azure::Core;
+
+namespace Azure { namespace Core { namespace Test {
+
+  class StringClient;
+
+  class StringOperation : public Operation<std::string> {
+
+  private:
+    StringClient* m_client;
+    std::string m_operationToken;
+    std::string m_value;
+
+  private:
```

Comment:
This `private` is redundant with line 20:
```suggestion
```

Reply:
An old habit of writing code where visibility was specified for members and functions with each being treated separately.